src/08-accessing.js

```diff
@@ -71,7 +77,10 @@ function getAddressStreet(person) {}
  *  getCountOfPhoneNumbers(person);
  *  //> 2
  */
-function getCountOfPhoneNumbers(person) {}
+function getCountOfPhoneNumbers(person) {
+
+  return person.numbers.push()
+}
```

Comment:
I believe this is one place where you are changing the object passed in to you, as we discussed. There's an array property that can tell you how many are in the array without _changing_ the array.